src/CRM/StateMachine.hs

```diff
@@ -25,6 +26,17 @@ data StateMachine input output where
     :: StateMachine a b
```

Comment:
Would it make sense to call this constructor `Sequence` or  `Sequential`?

Reply:
Sometimes I think about it, too. I'm not sure, though. `Compose` recalls a lot the terminology used in category theory. `Sequential` gives a more graphical idea how what's happening.

I'd stick with `Compose` for now. I'll ask to some more people what they think